README.md

```diff
@@ -1,2 +1,81 @@
-# CESM GMOM_JRA configuration
-CESM GMOM_JRA configuration (MOM6-CICE6-DATM-DROF-SLND-SWAV-SGLC) to run with payu
+# MOM6-CICE6-datm-drof configurations for ACCESS-OM3
+
+This repository contains several
+[ACCESS-OM3](https://github.com/COSIMA/access-om3) configurations using the following components:
+
+- [MOM6](https://github.com/mom-ocean/MOM6)
+- [CICE](https://github.com/CICE-Consortium/CICE)
+- [datm](https://github.com/ESCOMP/CDEPS) data model
+- [drof](https://github.com/ESCOMP/CDEPS) data model
```

Comment:
```suggestion
- [DATM](https://github.com/ESCOMP/CDEPS) data model
- [DROF](https://github.com/ESCOMP/CDEPS) data model
```

Reply:
or maybe link to docs?
https://escomp.github.io/CDEPS/versions/master/html/datm.html
https://escomp.github.io/CDEPS/versions/master/html/drof.html

Reply:
or the ACCESS-OM3 wiki, so we can centralise our user documentation
https://github.com/COSIMA/access-om3/wiki/Forcing-data-models